cesql_tck/like_expression.yaml

```diff
@@ -25,6 +28,12 @@ tests:
   - name: Percentage operator (6)
     expression: "'' LIKE 'abc'"
     result: false
+  - name: Percentage operator (7)
+    expression: "'.ab.cde.' LIKE '.%.%.'"
+    result: true
+  - name: Percentage operator (8)
+    expression: "'ab.cde' LIKE '.%.%.'"
```

Comment:
I'm curious, are you afraid that someone will treat "." as a special char?

Reply:
I am, because I was converting this to a regular expression in the ref implementation without escaping the dot :laughing: